packages/scandipwa/src/util/Request/QueryDispatcher.js

```diff
@@ -57,7 +59,10 @@ export class QueryDispatcher {
             this.promise = await executeGet(prepareQuery(queries), name, cacheTTL, this.controller.signal);
             this.onSuccess(this.promise, dispatch, options);
         } catch (err) {
-            this.onError(err, dispatch, options);
+            if (err.message !== REQUEST_ABORT_MESSAGE
+                && err.message !== REQUEST_ABORT_MESSAGE_2) {
+                this.onError(err, dispatch, options);
+            }
```

Comment:
Can we trust these strings? Can they change on different browsers and versions?

Reply:
Maybe there is some error code available?

Reply:
`REQUEST_ABORT_MESSAGE` as a substring of `REQUEST_ABORT_MESSAGE_2`.
So actually you don't need two constants, you can just search a substring.